renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+    "branchPrefix": "renovate-",
+    "prHourlyLimit": 0,
+    "lockFileMaintenance": {
+        "enable": true,
+        "schedule": [
+            "before 9am every weekday"
+        ],
+        "automergeType": "branch",
+        "automerge": true
+    },
+    "packageRules": [
+        {
+            "depTypeList": [
+                "devDependencies"
+            ],
+            "rangeStrategy": "replace",
+            "semanticCommitType": "Fix",
```

Comment:
Dev dependencies changes should not necessarily warrant a release, so this is probably best as `Other: `.